### PR TITLE
Bug 1702468 - Update private import paths on Javascript template

### PR DESCRIPTION
**Pull request by brizental**
### Pull Request checklist ###

- [x] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes a changelog entry to `CHANGELOG.md` or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

Friend PR for: https://github.com/mozilla/glean.js/pull/768

### CODE REVIEW

File: tests/test_javascript.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def test_duplicate(tmpdir):
         content = fd.read()
         assert (
             content.count(
-                'import CounterMetricType from "@mozilla/glean/webext/private/metrics/counter";'  # noqa
+                'import CounterMetricType from "@mozilla/glean/private/metrics/counter";'  # noqa
```
Comment (Dexterp37) [inline tests/test_javascript.py:242]: Why is this needed?

Comment (brizental) [reply tests/test_javascript.py:242]: I changed the way to build Glean.js, so that we don't need to duplicate the whole lib for each platform as before. So all the `private/` modules for all platforms are served from the same entry point (aside from Qt, of course). See https://github.com/mozilla/glean.js/pull/768

**Review by Dexterp37 — APPROVED** [review]
Please kindly add a changelog entry.
